src/api/client.js

```diff
@@ -73,6 +73,7 @@ export class Client {
     HTML: 'application/vnd.github.v3.html+json',
     JSON: 'application/vnd.github.v3+json',
     MERCY_PREVIEW: 'application/vnd.github.mercy-preview+json',
+    MOCKINGBIRD_PREVIEW: 'application/vnd.github.mockingbird-preview+json',
```

Comment:
we may want to somehow keep track of when these preview APIs end up changing

Reply:
Looks like this one opened in 2016, wow lol

https://developer.github.com/changes/2016-05-23-timeline-preview-api/

But still something to keep in mind 🤔

Reply:
The other preview API, MERCY_PREVIEW, which was for fetching topics of repository, was replaced by GraphQL. If no so-called preview versions in GraphQL, we can also switch the timeline API.